tools/swift-format/Sources/Rules/OperatorWhitespace.swift

```diff
@@ -1,15 +1,60 @@
-import Core
-import Foundation
-import SwiftSyntax
-
-/// Exactly one space must appear before and after each binary operator token.
-///
-/// Lint: If an invalid number of spaces appear before or after a binary operator, a lint error is
-///       raised.
-///
-/// Format: All binary operators will have a single space before and after.
-///
-/// - SeeAlso: https://google.github.io/swift#horizontal-whitespace
-public final class OperatorWhitespace: SyntaxFormatRule {
-
-}
+ import Core
+ import Foundation
+ import SwiftSyntax
+ 
+ /// Exactly one space must appear before and after each binary operator token.
+ ///
+ /// Lint: If an invalid number of spaces appear before or after a binary operator, a lint error is
+ ///       raised.
+ ///
+ /// Format: All binary operators will have a single space before and after.
+ ///
+ /// - SeeAlso: https://google.github.io/swift#horizontal-whitespace
+ public final class OperatorWhitespace: SyntaxFormatRule {
+  public override func visit(_ token: TokenSyntax) -> Syntax {
+    guard let nextToken = token.nextToken else { return token }
+    let exceptions =  ["...", "..<", ">.."]
+    
+    // Operators own their trailing spaces, so ensure it only has 1 space
+    // if there's another token in the same line.
+    if (token.tokenKind == .unspacedBinaryOperator(token.text) ||
+      token.tokenKind == .spacedBinaryOperator(token.text)),
+      !nextToken.leadingTrivia.containsNewlines,
+      !exceptions.contains(token.text){
```

Comment:
This can happen later, but it would be nice to see another set of diagnostics about removing spaces around the `...` and `..<` operators.